blocks/article-header/article-header.js

```diff
@@ -80,7 +80,7 @@ export default async function decorate(block) {
 
     if (articleHeaderData.caption) {
       const imageCaption = document.createElement('figcaption');
-      imageCaption.classList.add('util-detail-s', 'image-with-caption__caption');
```

Comment:
Necessary for the max-width `ch` value on the caption parent element to match the same as the body text size in the sections below, so these have the same width. I moved the font size styles to the children of this element with CSS instead of adding another wrapper div for the utility.